pkg/saver/saver.go

```diff
@@ -123,4 +133,6 @@ func Tar(src string, writers ...io.Writer) error {
 
 		return nil
 	})
+
+	return structure, err
```

Comment:
Suggestion：`return structure, nil`

Reply:
We cannot know if the err is nil or an error.